format-registry/stream/index-respec.html

```diff
@@ -151,14 +151,14 @@
               audio/mp4
               <br/>video/mp4
             </td>
-            <td><a href="mp4.html">ISO Common Encryption ('cenc') Protection Scheme for ISO Base Media File Format Stream Format</a></td>
+            <td><a href="https://www.w3.org/TR/eme-stream-mp4/">ISO Common Encryption ('cenc') Protection Scheme for ISO Base Media File Format Stream Format</a></td>
```

Comment:
Use the 'stream-registry-mp4' macro? Then we can clean up the URLs as appropriate. See the comment in the macro section.